src/LRS.js

```diff
@@ -158,6 +158,39 @@ TinCan client library
         },
 
         /**
+        Creates and returns a boundary for separating parts in
+        requests where the statement has an attachment
+
+        @method getBoundary
+        @private
```

Comment:
JS doesn't have actual private/public, etc. so properties/methods that are "private"ish should start with `_`, please rename these.